tests/wpt/metadata-css/css-transforms-1_dev/html/transform-input-018.htm.ini

```diff
@@ -1,5 +1,3 @@
 [transform-input-018.htm]
   type: reftest
-  expected:
-    if os == "mac": PASS
-    FAIL
+  expected: FAIL
```

Comment:
Could you add `bug: https://github.com/servo/servo/issues/10881` to both of these manifests as well?